src/transformers/modeling_tf_utils.py

```diff
@@ -343,14 +343,14 @@ class TFPreTrainedModel(tf.keras.Model, TFModelUtilsMixin, TFGenerationMixin):
           :class:`~transformers.PretrainedConfig` to use as configuration class for this model architecture.
         - **base_model_prefix** (:obj:`str`) -- A string indicating the attribute associated to the base model in
           derived classes of the same architecture adding modules on top of the base model.
-        - **authorized_missing_keys** (:obj:`List[str]`, `optional`) -- A list of re pattern of tensor names to ignore
+        - **_keys_to_ignore_on_load** (:obj:`List[str]`, `optional`) -- A list of re pattern of tensor names to ignore
```

Comment:
I think we should stop documenting this since it's private.

Reply:
makes sense - but they should be documented somewhere else instead, no?

Reply:
can there be a docstring section that is private?

Reply:
regardless, if we want to get it into v4, then probably remove those as you suggested and then think of where to document those now private keys later?